Sources/Preferences/PreferencesStyle.swift

```diff
@@ -1,6 +1,11 @@
 import Cocoa
 
 public enum PreferencesStyle {
+	public enum SegmentSize {
+		case fit
+		case uniform
```

Comment:
Have you considered a default `auto` size? Which would switch to fit if `uniform` cannot fit.

I don't think users ever want this:

<img width="620" alt="Screenshot 2020-02-22 at 14 11 03" src="https://user-images.githubusercontent.com/170270/75088259-467b9a80-557d-11ea-9c95-5914b7c6c73f.png">

So it kinda makes sense to do the right behavior by default.

Reply:
In `fit` mode, I could fit 5 segmented controls.

<img width="550" alt="Screenshot 2020-02-22 at 14 13 22" src="https://user-images.githubusercontent.com/170270/75088275-7e82dd80-557d-11ea-866b-fa0be80a3259.png">

What if the user needs more? Have you considered a mode that shrinks the text?

Kinda like the system about dialog:

<img width="623" alt="Screenshot 2020-02-22 at 14 13 48" src="https://user-images.githubusercontent.com/170270/75088280-9195ad80-557d-11ea-8b6d-5ff110fe15d7.png">

Reply:
I like the `.fitSmall` idea!

Recently, I've been thinking that we maybe should drop the `.uniform` size completely. I don't know if anyone actually prefers uniform sizes. 🤔

But _if_ someone really cares about uniform sizes, I'd like to think that they won't like the segment to change style to `.fit` for some of the tabs, and `.uniform` for others. The "right thing" here, in my opinion, would be to not encourage sloppy preference windows where the segments jiggle around when you switch the active page.

@sindresorhus WDYT?

Reply:
> Recently, I've been thinking that we maybe should drop the .uniform size completely. I don't know if anyone actually prefers uniform sizes. 🤔

I like the uniform style and I don't think we should get rid of it. It looks better than "fit" when there are just a few buttons. For example, in one of my apps, I use uniform style:

<img width="517" alt="Screenshot 2020-06-05 at 23 03 26" src="https://user-images.githubusercontent.com/170270/83892159-d3d7a900-a780-11ea-9bdf-3eac56a785d3.png">

> But if someone really cares about uniform sizes, I'd like to think that they won't like the segment to change style to .fit for some of the tabs, and .uniform for others. The "right thing" here, in my opinion, would be to not encourage sloppy preference windows where the segments jiggle around when you switch the active page.

Good point. Forgot about my idea about "auto". I didn't consider panes having different size.